docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) 
+that both have side effects on the environment, thus including the applications effects we'll have: 
```

Comment:
I would remove `, thus including the applications effects we'll have` to make easier to read.